sdk/src/main/java/io/dapr/client/DaprClientImpl.java

```diff
@@ -1175,6 +1178,51 @@ public Mono<UnsubscribeConfigurationResponse> unsubscribeConfiguration(Unsubscri
     }
   }
 
+  @Override
+  public <T> Mono<Void> scheduleJobAlpha1(Job<T> job) {
+    try {
+  	  final String name = job.getName();
+  	  final T data = job.getData();
+  
+  	  if (name == null || name.trim().isEmpty()) {
+  		  throw new IllegalArgumentException("Job name cannot be null or empty");
+  	  }
+  	  if (data == null) {
+  		  throw new IllegalArgumentException("Job data cannot be empty");
+  	  }
+  
+  	  DaprProtos.Job.Builder jobBuilder = DaprProtos.Job.newBuilder()
+  			  .setName(name);
+  	  if (data instanceof Message) {
+  		  jobBuilder.setData(Any.pack((Message)job.getData()));
+  	  } else {
+  		  jobBuilder.setData(Any.newBuilder().setValue(ByteString.copyFrom(this.objectSerializer.serialize(data))));
+  	  }
```

Comment:
@artursouza is this implementation fine?

Reply:
@artursouza the data of the Job can be something different than `Message`?

Reply:
@charan2628 @salaboy and @artursouza I see that `Job` `data` is modeled as `google.protobuf.Any`. In other places where we use `google.protobuf.Any` like `TransactionalActorStateOperation`, we check that `data` is either `String` or `byte[]`. I think we should stick to this approach.

We don't want to expose `com.google.protobuf.Message` to `DaprClient` users, otherwise they will have too many options to create a `Job`, while having `data` as either `String` or `byte[]` covers most of the use cases.

Reply:
@salaboy @artur-ciocanu is this implementation fine?